pkg/api/types.go

```diff
@@ -683,6 +683,14 @@ const (
 	AffinityTypeNone AffinityType = "None"
 )
 
+// LoadBalancerInfo defines whether the LoadBalancer is rewriting or
+// pass through
+type LoadBalancerInfo struct {
+	DestIP   string `json:"destIP,omitempty" description:"External IP of the Load Balancer"`
+	SourceIP string `json:"sourceIP,omitempty" description:"Source IP of the Load Balancer"`
+	Rewrite  bool   `json:"rewrite,omitempty" description:"Does the LB rewrite packets"`
```

Comment:
If we're going to do this - and I still need to think about it some more - we should define this as a string type and define "Forwarding" and "BackToBack" for GCE and Rackspace respectively.

@brendanburns we talked about AWS, but I forgot what you said - thoughts on this?